src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -713,7 +712,7 @@ def write_segmentation(
             )
 
             plane_segmentation_metadata = metadata["Ophys"]["ImageSegmentation"]["plane_segmentations"]
-            plane_segmentation_name = plane_segmentation_metadata[plane_no_loop]["name"]
+            plane_segmentation_name = plane_segmentation_metadata[0]["name"]
```

Comment:
not sure this is the best fix here

Reply:
I think is fine temporary. Let's discuss this on the meeting what to do with this MultiSegmentationExtractor case. The metadata as a list is not consistent with the workflow in all the other pipelines.